lib/scholar/neighbors/random_projection_forest.ex

```diff
@@ -0,0 +1,363 @@
+defmodule Scholar.Neighbors.RandomProjectionForest do
+  @moduledoc """
+  Random Projection Forest.
```

Comment:
```suggestion
  Random Projection Forest for approximate k-nearest neighbors.
```

Reply:
Or alternatively:

```suggestion
  Random Projection Forest for (approximate) nearest neighbor searches.
```

Or similar. :D